modules/ROOT/pages/spring/spring-module.adoc

```diff
@@ -103,19 +122,19 @@ You can use Spring objects in Mule configuration components, for instance, to po
         http://www.mulesoft.org/schema/mule/db
         http://www.mulesoft.org/schema/mule/db/current/mule-db.xsd">
 
-    <spring:config name="springConfig" files="beans.xml" doc:id="8623553f-16f6-45d9-acef-e23764e05bf8" />
+    <spring:config name="springConfig" files="beans.xml" />
 
-    <db:config name="derbyConfig" doc:name="Database Config" doc:id="06993a86-2062-4c9a-b9a9-a663ff6b2eed" >
+    <db:config name="derbyConfig" doc:name="Database Config"  >
         <db:derby-connection database="datasource" create="true" />
     </db:config>
 </mule>
 ----
 
-== Use Spring Properties Placeholders
+== Spring Properties Placeholders
 
 Properties defined by Spring properties placeholders cannot be used within the Mule app configuration, only in Spring configuration files. However, properties defined in Mule through `<configuration-properties/>` can be used within Spring configuration files.
 
-== Limitations over spring objects
+== Limitations of Spring Objects
 
 * Spring objects cannot implement Mule API lifecycle methods.
 * Spring objects cannot be injected with Mule API services. The only service that can be used is the `org.mule.runtime.api.artifact.Registry`, which can be used to access any other service within the Mule API. This can be accomplished by using `@PostConstruct` over a method to initialize the object and retrieve the Mule API service.
```

Comment:
Limit list items to one sentence, with additional detail following in a para below:
```suggestion
* Spring objects cannot be injected with Mule API services.
The only service that can be used is the `org.mule.runtime.api.artifact.Registry`, which can be used to access any other service within the Mule API. This can be accomplished by using `@PostConstruct` over a method to initialize the object and retrieve the Mule API service.
```

Reply:
HIDDEN TEXT BELOW
Edits to narrative:

== Use Spring Security as a Security Manager

The Spring module supports the use of Spring security as a security manager in Mule apps.
You must define an authentication manager in the Spring configuration file:

[source,xml,linenums]

<ss:authentication-manager alias="authenticationManager">
ss:authentication-provider
<ss:user-service id="userService">
<ss:user name="admin" password="admin" authorities="ROLE_ADMIN" />
<ss:user name="joe" password="secret" authorities="ROLE_ADMIN" />
<ss:user name="anon" password="anon" authorities="ROLE_ANON" />
<ss:user name="user" password="password" authorities="ROLE_ANON" />
<ss:user name="ross" password="ross" authorities="ROLE_ANON" />
<ss:user name="marie" password="marie" authorities="ROLE_ANON" />
</ss:user-service>
</ss:authentication-provider>
</ss:authentication-manager>